source/index.js

```diff
@@ -36,15 +37,32 @@ var likely = function (node, options) {
 };
 
 /**
+ * Will be deprecated in 3.0
+ * @returns {Likely}
+ */
+var likely = function () {
+    // eslint-disable-next-line no-console
+    console.warn('This function will be DEPRECATED in 3.0. Use likely.initiate instead.');
+    return likely.initiate.apply(likely, arguments);
+};
+
+/**
  * Initiate Likely buttons on load
+ * @param {Node|Object} [node] a particular node of widget,
+ *                             if not specified,
+ *                             tries to init all the widgets
  * @param {Object} [options] additional options for each widget
  */
-likely.initiate = likely.initate = function (options) {
-    var widgets = dom.findAll('.' + config.name);
+likely.initiate = likely.initate = function (node, options) {
+    var nodes = node instanceof Node ? [node] : dom.findAll('.' + config.name);
 
-    utils.toArray(widgets)
-        .forEach(function (widget) {
-            likely(widget, options);
+    if (!(node instanceof Node)) {
+        // eslint-disable-next-line no-param-reassign
+        options = node;
+    }
```

Comment:
> I'm not sure hot to make it right

Do you mean you aren’t sure about `options = node`? I think it’s OK. Or are you about something else?